src/sonic_exporter/__init__.py

```diff
@@ -1,10 +0,0 @@
-import logging
```

Comment:
@Cellebyte 
__init__.py is a better place to initiate/config logging like services. 
__init__.py will always be executed if some one uses sonic-exporter in their project deps.